flask/auth.py

```diff
@@ -28,16 +28,12 @@ def generate_uuid():
 @app.route("/auth/login", methods=["POST"])
 @swag_from("documentation/login.yaml", methods=["POST"])
 def login():
-    uuid = request.form.get("uuid")
-    if not uuid:
-        email = request.form.get("email")
-        password = request.form.get("password")
-        password += salt
-        hashed_password = hashlib.sha256(password.encode()).hexdigest()
-        user = db.users.find_one({"email": email, "password": hashed_password})
-        uuid = generate_uuid() if user['loggedCount']==0 else user['uuid']
-    else:
-        user = db.users.find_one({"uuid": uuid})
+    email = request.form.get("email")
+    password = request.form.get("password")
+    password += salt
+    hashed_password = hashlib.sha256(password.encode()).hexdigest()
+    user = db.users.find_one({"email": email, "password": hashed_password})
+    uuid = generate_uuid() if user['loggedCount']==0 else user['uuid']
```

Comment:
Backend will crash when new user directly tries to login and this statement user["loggedCount"] will result in None error. Because it is trying to access metadata from NoneType.

Reply:
It would not crash as  I have already added https://github.com/henilp105/registry/blob/main/flask/auth.py#L131 the increment for a new user to increment `loggedCount`.

Reply:
You can reproduce it by entering either wrong email or password in the ``/auth/login`` API.